src/main/java/in/zapr/druid/druidry/query/config/Interval.java

```diff
@@ -28,18 +29,20 @@
 @EqualsAndHashCode
 public class Interval {
 
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+
     private final static String DRUID_INTERVAL_FORMAT = "%s/%s";
 
-    private DateTime startTime;
-    private DateTime endTime;
+    private ZonedDateTime startTime;
+    private ZonedDateTime endTime;
 
-    public Interval(@NonNull DateTime startTime, @NonNull DateTime endTime) {
+    public Interval(@NonNull ZonedDateTime startTime, @NonNull ZonedDateTime endTime) {
         this.startTime = startTime;
         this.endTime = endTime;
     }
 
     @JsonValue
     private String getIntervalAsString() {
-        return String.format(DRUID_INTERVAL_FORMAT, startTime.toDateTimeISO(), endTime.toDateTimeISO());
+        return String.format(DRUID_INTERVAL_FORMAT, FORMATTER.format(startTime), FORMATTER.format(endTime));
```

Comment:
Here also there will be contract issue and library might be giving wrong interval/s than what user intended as we are not including offset which was getting included before with `toDateTimeISO()`.

Eg : Now `getIntervalAsString()` would be returning `2020-07-12T22:55:15.714Z/2020-07-12T22:55:15.714Z` vs previous value `2020-07-12T22:55:15.714+05:30/2020-07-12T22:55:15.714+05:30` with offsets.